./Archives of Biochemistry and Biophysics(all years)/21001-21500.txt	string index out of range
./human reproduction (1959-2013)/1959-1994/savedrecs (3).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/1995-2005/savedrecs (11).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/1995-2005/savedrecs (14).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/1995-2005/savedrecs (17).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/1995-2005/savedrecs (3).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/2006-2013/savedrecs (1).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/2006-2013/savedrecs (13).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/2006-2013/savedrecs (4).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/2006-2013/savedrecs (6).txt	'NoneType' object has no attribute '__getitem__'
./human reproduction (1959-2013)/2006-2013/savedrecs (9).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2000-2004/savedrecs (2).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2000-2004/savedrecs (20).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2000-2004/savedrecs (3).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2000-2004/savedrecs.txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2005-2009/savedrecs (11).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2005-2009/savedrecs (13).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2005-2009/savedrecs (16).txt	'NoneType' object has no attribute '__getitem__'
./nature (1959-2013)/2005-2009/savedrecs (22).txt	'NoneType' object has no attribute '__getitem__'
./New Biotechnology/1501-2000.txt	'NoneType' object has no attribute '__getitem__'
./New Biotechnology/501-1000.txt	'NoneType' object has no attribute '__getitem__'
./plos one (1955-2015)/35001-35500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/instance_metadatum/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10bac6350>: Failed to establish a new connection: [Errno 60] Operation timed out',))
./plos one (1955-2015)/3501-4000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10d219d10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/35501-36000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10caf6250>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/36001-36500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10bea8910>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/36501-37000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10ee2f6d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/37001-37500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x108be0550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/37501-38000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10975e490>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/38001-38500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10eed1e90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/38501-39000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x11f4d3b90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/39001-39500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x108e8f310>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/39501-40000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10fae2410>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/40001-40500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10dbd8050>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/4001-4500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x118acbe10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/40501-41000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10a323690>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/41001-41500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10bd2a190>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/41501-42000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10cccee50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/42001-42500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x11172bfd0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/42501-43000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x1187e62d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/43001-43500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10b07d550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/43501-44000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10d36be50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/44001-44500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10baf0890>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/44501-45000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10cb5d810>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/45001-45500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10e868410>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/4501-5000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10bed3190>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/45501-46000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10fdcf990>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/46001-46500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x110b4ae10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/46501-47000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10d2550d0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/47001-47500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10be9aad0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/47501-48000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x110ff5450>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/48001-48500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x1099e8e10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/48501-49000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10901ca10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/49001-49500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x112e8c210>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/49501-50000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10a5dc810>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/50001-50500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x108e3d810>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/5001-5500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x109857b50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/501-1000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10e896f90>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/50501-51000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x109f8b750>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/51001-51500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x115585f10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/51501-52000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x110309550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/52001-52500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10dab4ed0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/52501-53000.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x10bffb850>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./plos one (1955-2015)/53001-53500.txt	HTTPConnectionPool(host='diging.asu.edu', port=80): Max retries exceeded with url: /tethne/rest/paper_instance/ (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x110b17e10>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
./Proceedings of the National Academy of Sciences of the United States of America(1955-2015)/92001-92500.txt	sequence item 0: expected string, list found
./Science (1955-2015)/2005-2009/savedrecs (10).txt	'NoneType' object has no attribute '__getitem__'
./Science (1955-2015)/2005-2009/savedrecs (15).txt	'NoneType' object has no attribute '__getitem__'
./Science (1955-2015)/2005-2009/savedrecs (21).txt	'NoneType' object has no attribute '__getitem__'
./Science (1955-2015)/2005-2009/savedrecs (3).txt	'NoneType' object has no attribute '__getitem__'
./Science (1955-2015)/2010-2013/savedrecs (5).txt	'NoneType' object has no attribute '__getitem__'
